docs/source/inquiries.rst

```diff
@@ -0,0 +1,334 @@
+    | parent   | 59ab26af32ed3575803bf139                                     |
+    | roles    |                                                              |
+    | users    | [                                                            |
+    |          |     "testu"                                                  |
+    |          | ]                                                            |
+    | tag      | developers                                                   |
+    | ttl      | 1440                                                         |
+    | schema   | {                                                            |
+    |          |     "type": "object",                                        |
+    |          |     "properties": {                                          |
+    |          |         "continue": {                                        |
+    |          |             "type": "boolean",                               |
+    |          |             "description": "Would you like to continue the   |
+    |          | workflow?"                                                   |
+    |          |             "required": true
+    |          |         }                                                    |
+    |          |     },                                                       |
+    |          |     "title": "response_data"                                 |
+    |          | }                                                            |
+    +----------+--------------------------------------------------------------+
+
+In this view, we see the schema in use requires a single key: ``continue``, whose value must be boolean. Fortunately, the ``st2`` client makes this really easy; when you run the command ``st2 inquiry respond <inquiry id>``, it will step through each of these values, prompting you with the provided description, and giving a hint as to what type is expected. You simply respond to each prompt:
+
+.. code-block:: bash
+
+    ~$ st2 inquiry respond 59ab26af32ed35752062d2dc
+    Should we continue? (boolean): True
+
+     Response accepted. Successful response data to follow...
+    +----------+--------------------------+
+    | Property | Value                    |
+    +----------+--------------------------+
+    | id       | 59ab26af32ed35752062d2dc |
+    | response | {                        |
+    |          |     "continue": true     |
+    |          | }                        |
+    +----------+--------------------------+
+
+It's very important that each property in the response schema has a proper description, as shown in the default example, as this is what prompts the user for required values when it's time to respond.
+
+If you provide a value that doesn't conform to the provided schema, an error will be shown, and the workflow will remain paused:
+
+.. code-block:: bash
+
+    ~$ st2 inquiry respond 59ab26af32ed35752062d2dc
+    Should we continue? (boolean): 123
```

Comment:
This behavior is not what I see

```
(virtualenv)vagrant@st2dev /m/s/s/st2 ❯❯❯ st2 inquiry respond 59ca9c86d9d7ed1559489a20                                                                                                       api-ask-response ✭ ◼
usage: st2 inquiry respond [-h] [-t TOKEN] [--api-key API_KEY] [-j] [-y]
                           id response
st2 inquiry respond: error: too few arguments
(virtualenv)vagrant@st2dev /m/s/s/st2 ❯❯❯
```

Reply:
Ack, the docs are pretty far behind the actual implementation at this point, esp in the client. Am addressing that now actually

Reply:
Per slack, this behavior changed in a recent commit, so using an updated branch should better reflect this portion of the docs.

Reply:
Ignore me. I was testing with an older version.